apps/i18n/common/en_us.json

```diff
@@ -645,9 +645,9 @@
   "feedbackLoadError": "There's been an error establishing a connection to our servers. Please refresh the page and try again.",
   "feedbackNoneYet": "You do not have any feedback on any lessons from your teacher(s) yet.",
   "feedbackNotSavedWarning": "Are you sure? Your feedback may not be saved.",
-  "feedbackNotification": "Your teacher has left you feedback!",
+  "feedbackNotification": "Your teacher left you feedback.",
   "feedbackNotificationButton": "View feedback",
-  "feedbackNotificationDetails": "You have {numFeedbackLevels} levels with unread feedback.",
+  "feedbackNotificationDetails": "You have new feedback on {numFeedbackLevels} level(s).",
```

Comment:
there is some built-in logic that lets us handle this case by pluralizing the phrase:

https://github.com/code-dot-org/code-dot-org/blob/6a96a7c967624de49b9367d33a59a29c76c0e194/apps/i18n/common/en_us.json#L612

Reply:
what?! thanks for the tip!